client/packages/requisitions/src/ResponseRequisition/DetailView/ResponseLineEdit/ResponseLineEdit.tsx

```diff
@@ -229,8 +229,15 @@ export const ResponseLineEdit = ({
                 <NumericTextInput
                   width={INPUT_WIDTH}
                   value={draft?.requestedQuantity}
-                  onChange={value => update({ requestedQuantity: value })}
+                  onChange={value => {
+                    if (draft?.suggestedQuantity === value) {
+                      update({ requestedQuantity: value, reason: null });
+                    } else {
+                      update({ requestedQuantity: value });
+                    }
+                  }}
                   disabled={!!hasLinkedRequisition}
+                  onBlur={save}
```

Comment:
Missed save on blur so this value wasn't being saved if it was the only one edited..